### PR TITLE
docs(common): contributing guide

### PR DESCRIPTION
**Pull request by eunjae-lee**
## What?

This PR updates the CONTRIBUTING.md 

[View the rendered file →](https://github.com/storyblok/field-plugin/blob/EXT-1256-contributing-readme/CONTRIBUTING.md)

## Why?

JIRA: EXT-1256

### CODE REVIEW

**Comment by eunjae-lee** [comment]
Hi @BibiSebi and @demetriusfeijoo 
This is a new CONTRIBUTING.md guide for those who want to contribute to this SDK. Feel free to leave lots of questions, comments, and suggestions here 🙌🏻

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,19 +1,155 @@
...
+Then, go to [Releases](https://github.com/storyblok/field-plugin/releases) and create a new release. The title should follow the format `<PACKAGE-NAME>@<VERSION>`. Examples:
+
+- `@storyblok/field-plugin@0.0.1`
+- `@storyblok/field-plugin-cli@1.0.0-beta.2`
+
+You can find a sample release [here](https://github.com/storyblok/field-plugin/releases/tag/%40storyblok%2Ffield-plugin-cli%400.0.1-beta.2). Make sure you write accurate changelogs, ensuring that library changes are not included in a CLI release and vice versa.
+
+Once a release is created, one of the two GitHub Actions - [.github/workflows/npm-publish-library.yml](https://github.com/storyblok/field-plugin/blob/main/.github/workflows/npm-publish-library.yml) or [.github/workflows/npm-publish-cli.yml](https://github.com/storyblok/field-plugin/blob/main/.github/workflows/npm-publish-cli.yml) - will run and deploy it to NPM.
+
+### Order of releases
```
Comment (BibiSebi) [inline CONTRIBUTING.md:153]: Should we also add a section explaining what is important for the PR. Like explaining what has been changes and some guidelines to not submit a huge PR or something similar

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,19 +1,155 @@
...
+- Naming `gitignore` files without the dot (`gitignore`) as otherwise the file will be ignored by `npm publish`.
+- Omitting `.env.local.example` as it will be copied from the `monorepo` template.
+
+### Helpers
+
+This section will be filled once the helpers are shipped. Work is in progress.
+
+## Release
+
+You can release either `@storyblok/field-plugin` or `@storyblok/field-plugin-cli`. To begin the release process, run the following command on the `main` branch:
```
Comment (BibiSebi) [inline CONTRIBUTING.md:134]: should contributors have the right to release?

**Comment by eunjae-lee** [comment]
@BibiSebi thanks for the review! I've addressed your feedback :) 
https://github.com/storyblok/field-plugin/pull/225/commits/9f42d8d0a6017dc6bda207733c8b68da1b1abd05

**Review by BibiSebi — APPROVED** [review]
Awesome job @eunjae-lee 👏 let's merge it and see if there is any feedback once someone starts contributing

**Pull request merged**